scala/tempo/build.sbt

```diff
@@ -9,6 +9,8 @@ libraryDependencies ++= Seq(
   "org.apache.spark" %% "spark-core" % "3.0.0",
   // https://mvnrepository.com/artifact/org.apache.spark/spark-sql_2.11
   "org.apache.spark" %% "spark-sql" % "3.0.0",
+  "org.apache.spark" %% "spark-mllib" % "3.0.0",
+  "org.scalactic" %% "scalactic" % "3.0.1" % "test",
```

Comment:
Curious why this is included - I didn't notice this package being referenced in the test code.

Reply:
I was testing out Scalactic - forgot to remove it from the build file. I'll take it out for now (but we may want it back in later)

Reply:
This is done.